packages/components/src/Toast/Toast.tsx

```diff
@@ -102,9 +102,9 @@ export function Toast({
   }
 
   function getTimeout() {
-    const min = 2000;
-    const max = 5000;
-    const time = message.length * 100;
+    const min = 5000;
+    const max = 10000;
+    const time = message.length * 200;
```

Comment:
Perhaps too much of a smartboi comment (sorry!) figure I'll ask anyways.  Should time here also account for something like the variation type as well?

Reply:
If the reasoning is like "stick around longer for error"...

TLDR we should actually remove error as an available Toast type and only use Banner for errors (but that's a separate issue, imo)

Reply:
Okay gotcha!  Ya, not even an issue here, just figured I'd ask if something like `info` should stick around for slightly longer than `success`/`default`